api/src/main/java/org/openmrs/Encounter.java

```diff
@@ -353,6 +355,36 @@ public void removeObs(Obs observation) {
 	public void setOrders(Set<Order> orders) {
 		this.orders = orders;
 	}
+
+	/**
+	 * @return Returns the order groups
+	 */
+	public Set<OrderGroup> getOrderGroups() {
+		if (orderGroups == null) {
+			orderGroups = new LinkedHashSet<OrderGroup>();
```

Comment:
Not Applicable:

We have removed Set<OrderGroups> from Encounter as a change from the latest discussion